engine/function/src/templates/Numeric.ftl

```diff
@@ -1588,6 +1588,73 @@ public class Numeric {
         return product(new ${pt.vectorDirect}(values));
     }
 
+    /**
+     * Returns the differences between elements in the input vector separated by a stride.
+     * A stride of k returns v(i)=e(i+k)-e(i), where v(i) is the ith computed value e(i) is the ith input value.
```

Comment:
Is there a connecting word missing here before e(i)? like "and"?

where v(i) is the ith computed value, and e(i) is the ith input value.